tools/iothub-explorer/iothub-explorer.js

```diff
@@ -143,27 +143,23 @@ else if (command === 'list') {
   });
 }
 else if (command === 'create') {
-  if (!arg1) inputError('No device information given');
-  var info;
-  try {
-    // 'create' command expects either deviceId or JSON device description
-    info = (arg1.charAt(0) !== '{') ? { "deviceId": arg1 } : JSON.parse(arg1);
+  createDevice(arg1);
+}
+else if (command === 'import'){
+  if(!arg1) inputError('Import file not specified');
+  
+  if(!fs.existsSync(arg1)) {
+    inputError("Import file does not exist");
   }
-  catch(e) {
-    if (e instanceof SyntaxError) inputError('Device information isn\'t valid JSON');
-    else throw e;
+  
+  var data = fs.readFileSync(arg1, 'utf-8');
+  
+  var deviceIds = data.split('\n');
+  
+  for(var i=0; i<deviceIds.length; i++){
+    var deviceId = deviceIds[i].replace('\r','');
+    createDevice(deviceId);
```

Comment:
This code looks correct, although I'd prefer to use more common (and succinct) JavaScript patterns:
```
deviceIds.forEach(function (id) {
  createDevice(id.trim());
});
```